code42_util.py

```diff
@@ -40,4 +40,5 @@ def build_alerts_query(
         filters.append(Severity.is_in(severities))
     filters.append(build_date_range_filter(DateObserved, start_date, end_date))
     query = AlertQuery.all(*filters)
+    query.sort_direction = "asc"
```

Comment:
@peterbriggs42 @timabrmsn  is this change okay with both of you??
We should be getting the earlier polls first, it makes more sense
(polling was not working before because of this + that we get the last alert timestamp)

Reply:
yep, makes sense!